### DISCUSSION TITLE
Is there a way to see who is connected to realtime?

### DISCUSSION THREAD

**Discussion by mmattbtw** [pocketbase/pocketbase#1607] [Q&A] [answered]
I'd like to be able to see how many people are connected to the realtime database, and maybe display it on the app too.

**Comment by ganigeorgiev** [comment, marked as answer]
No, there is no builtin "presence"-like functionality in PocketBase because it is not clear who'll have access to this information, whether it needs to be grouped/aggreagated, what would be the criteria for that, etc. (you can also check https://github.com/pocketbase/pocketbase/discussions/845).

You can implement it on your own if you are [using PocketBase as framework](https://pocketbase.io/docs/use-as-framework/) and bind to the available [Realtime API event hooks](https://pocketbase.io/docs/event-hooks/#realtime-api-hooks).

**Reply by ganigeorgiev** [reply]
_I'll consider documenting it better in the future, but you can access all realtime connected clients and their subscriptions by using `app.SubscriptionsBroker().Clients()` (godoc: https://pkg.go.dev/github.com/pocketbase/pocketbase@v0.11.2/tools/subscriptions)._
